src/main/java/com/notably/logic/suggestion/handler/OpenSuggestionArgHandler.java

```diff
@@ -53,6 +59,7 @@ public OpenSuggestionArgHandler(Model model, CorrectionEngine<AbsolutePath> path
         }
 
         if (title.isEmpty()) {
+            logger.warning("title is empty");
```

Comment:
This shouldn't be a warning? Since it's a suggestion and it's not really an invalid thing to have a blank title